include/linux/soundwire/sdw.h

```diff
@@ -366,6 +369,8 @@ struct sdw_slave_prop {
 	u32 quirks;
 	bool clock_reg_supported;
 	bool use_domain_irq;
+	u32 sdca_interrupt_register_list;
```

Comment:
I would recommend to run `pahole` and perhaps rearrange the fields to remove unnecessary gaps and reduce memory footprint.

Reply:
done